tox.ini

```diff
@@ -1,8 +1,10 @@
 [tox]
 envlist =
-    py{2,27}-sphinx{16,17,18,20,21,22,23,24,30,31,32,33,34,35,40,41,42,43,44,45,50,51}{-html4,-html5,}{-qa,}
-    py{3,36,37,38,39}-sphinx{16,17,18,20,21,22,23,24,30,31,32,33,34,35,40,41,42,43,44,45,50,51,latest}{-html4,-html5,}{-qa,}
-    py{310}-sphinx{42,43,44,45,50,51,latest}{-html4,-html5,}{-qa,}
+    # Python 2.7 support was removed in Sphinx 2
+    py{2,27}-sphinx{16,17,18}{-html4,-html5,}{-qa,}
+    py{3,36,37,38,39}-sphinx{16,17,18,20,21,22,23,24,30,31,32,33,34,35,40,41,42,43,44,45,50,51,52,latest}{-html4,-html5,}{-qa,}
```

Comment:
Why do we have `py3` and `py3x`? What py3 means in this context? (the same question applies for `py2`)

Reply:
I actually have no idea! I left it there from earlier. I could imagine it means "run whatever is default on the system".

But it doesn't make any sense in the Circle CI setup, nor if running Tox locally, I'll remove it. Thanks for noticing :+1: